com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/util/ExecutionRetrier.java

```diff
@@ -47,7 +53,14 @@ public ExecutionRetrier withWaitTimeBetweenRetriesInMillis(long waitTimeBetweenR
             }
             sleep(waitTimeBetweenRetriesInMillis);
         }
-        return supplier.get();
+        try {
+            return supplier.get();
+        } catch (Exception e) {
```

Comment:
This is a bit inconsistent. The try-catch block above retries on ResourceAccessException and CloudOperationException, but this ignores every type of exception. What if the 2nd retry throws an IOException? Why would the last retry be resistant to IOExceptions, but the second wouldn't?